.travis.yml

```diff
@@ -166,6 +166,7 @@ jobs:
         - COMPILER="ccache clang++-3.7 -Qunused-arguments -fcolor-diagnostics"
         - CCACHE_CPP2=yes
         - EXTRA_CXXFLAGS="-DDEBUG"
+        - perl -p -i -e 's/^#define USE_DSTRING$//' src/util/irep_ids.h
```

Comment:
Since this perl is rather ugly, is it worth tweaking the `#define USE_DSTRING` (see my other comment on src/util/irep_ids.h) to avoid needing to do this perl hack?